### DISCUSSION TITLE
on-created-empty window not focused

### DISCUSSION THREAD

**Discussion by tarpan** [hyprwm/Hyprland#6037] [Q&A]
Hi, after the upgrade to hyprland v. 0.40.0 windows opened with the "on-created-empty" workspace option are no longer focused. This is irritating, as I have to move my mouse every time on such occasions for the window to get focus. It didn't behave like this in the previous versions. Can I change this behaviour in the config?

**Comment by ab1nn** [comment]
Any fix? The programs are not spawning in that specific workspace as well since update. Maybe since the focus is on some other workspace. This is really a pain.

**Comment by hiddenruins** [comment]
Set the `initial_workspace_tracking` to 0. 

```
misc {
  initial_workspace_tracking = 0
}
```

**Reply by 4aykovski** [reply]
it's worked

**Comment by sungyoonc** [comment]
Try git. Should be fixed with #7657. lmk if focus issue isn't fixed after this MR
